apps/frontend/src/views/Checklist.vue

```diff
@@ -0,0 +1,312 @@
+<template>
```

Comment:
we should have the two snackbar notifications pop up that say that one should load in a checklist and (on deselection of a checklist) say that you could select/load in a checklist similar to the results page

Reply:
Added